fastpair/test/test_fastpair.py

```diff
@@ -220,9 +216,9 @@ def test_update_point(self, PointSet):
         assert old not in fp
         assert new in fp
         assert len(fp) == len(ps)  # Size shouldn't change
-        l = [(fp.dist(a, b), b) for a, b in zip(cycle([new]), ps)]
-        res = min(l, key=itemgetter(0))
-        neigh = fp.neighbors[new]
+        l_ = [(fp.dist(a, b), b) for a, b in zip(cycle([new]), ps)]
+        res = min(l_, key=itemgetter(0))  # noqa: F841
```

Comment:
I wasn't sure if `res` and `neigh` were actually needed for something, but not being testing so I left in and added logic for `ruff` to ignore.

Reply:
they _were_ used in the following asserts, but have been commented out for reasons I am no longer aware of :P

Reply:
So we could either turn those back on, or just drop res and neigh I suspect.